src/Sentry/Android/JavaLogger.cs

```diff
@@ -0,0 +1,21 @@
+namespace Sentry.Android
+{
+    internal class JavaLogger : JavaObject, Java.ILogger
+    {
+        private readonly SentryOptions _options;
+
+        public JavaLogger(SentryOptions options) => _options = options;
+
+        public void Log(Java.SentryLevel level, string message, JavaObject[]? args) =>
+            _options.DiagnosticLogger?.Log(level.ToSentryLevel(), message, null, args?.Cast<object?>());
+
+        public void Log(Java.SentryLevel level, string message, Throwable? throwable) =>
+            _options.DiagnosticLogger?.Log(level.ToSentryLevel(), message, throwable);
+
+        public void Log(Java.SentryLevel level, Throwable? throwable, string message, params JavaObject[]? args) =>
```

Comment:
isnt the `?` on `params JavaObject[]? args`  redundant?

Reply:
Possibly, but it's on the auto generated `Java.ILogger` interface, so it has to be here too or it fails with a compilation error.